go_modules/spec/dependabot/go_modules/file_fetcher_spec.rb

```diff
@@ -6,6 +6,12 @@
 require_common_spec "file_fetchers/shared_examples_for_file_fetchers"
 
 RSpec.describe Dependabot::GoModules::FileFetcher do
+  after do
+    FileUtils.rm_rf(repo_contents_path)
+  end
+
+  after { FileUtils.rm_rf(repo_contents_path) }
+
```

Comment:
```suggestion
```

The same `after` block appears twice here.

Reply:
it's the blank line that was needed.
 rubocop fixes.

Reply:
I'm not sure I follow. You have the same setup code appearing twice in this file, so I was suggesting removing it.